src/core/runner.vala

```diff
@@ -0,0 +1,13 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private interface Games.Runner : Object {
+	public abstract Gtk.Widget? get_display ();
```

Comment:
And what if I still want to run my SNES games with say ZSNES?
I really think `get_display()` is too much for `Games.Runner`.